sormas-api/src/main/resources/captions_de-DE.properties

```diff
@@ -145,6 +145,7 @@ ActivityAsCase.activityAsCaseType=Art der Aktivität
 ActivityAsCase.activityAsCaseTypeDetails=Art der Aktivität Details
 ActivityAsCase.location=Ort
 ActivityAsCase.typeOfPlace=Art der Einrichtung
+ActivityAsCase.typeOfPlaceIfSG=Einrichtung (IfSG)
```

Comment:
See above; also, adding German translations is not necessary in general because those will be overridden by Crowdin with the English default anyway (until there's a translation in Crowdin)

Reply:
See above.

Reply:
See above.